NEWS.rst

```diff
@@ -0,0 +1,6 @@
+3.1.1.dev18+g17f6470 (2017-05-23)
+==================
+
```

Comment:
Shouldn't this file be `CHANGELOG.rst` given the configuration in `pyproject.toml`?